<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图像批量拼接器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startTrimPushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>开始拼接</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>分辨率与画质</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>50</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>高</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="WidthLineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>3000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>50</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>宽</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="HeightLineEdit">
      <property name="geometry">
       <rect>
        <x>252</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>2000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="whiteLineWidthLineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>180</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>白色边框宽度</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="imgQualitySpinBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>21</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>80</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>20</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>图像质量
(0-100)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>图像拼接规格</string>
     </attribute>
     <widget class="QLineEdit" name="FrameWLineEdit">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>198</x>
        <y>90</y>
        <width>50</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>高</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>90</y>
        <width>50</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>宽</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FrameHLineEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="ForceResizeCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>18</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>强制拉伸</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
